source/ed/connectors/fusio_parser.cpp

```diff
@@ -1545,14 +1545,22 @@ void GridCalendarTripExceptionDatesFusioHandler::handle_line(Data&, const csv_ro
 
 void AdminStopAreaFusioHandler::init(Data& data) {
     admin_c = csv.get_pos_col("admin_id");
-    stop_area_c = csv.get_pos_col("station_id");
-    for (const auto& object_code_map : data.object_codes) {
-        for (auto& object_code : object_code_map.second) {
-            if (object_code_map.first.type == nt::Type_e::StopArea && object_code.first == "external_code") {
-                const auto stop_area = gtfs_data.stop_area_map.find(object_code_map.first.pt_object->uri);
-                if (stop_area != gtfs_data.stop_area_map.end()) {
-                    for (const auto& external_code : object_code.second) {
-                        tmp_stop_area_map[external_code] = stop_area->second;
+
+    stop_area_c = csv.get_pos_col("stop_id");
+
+    // For retro compatibity
+    // TODO : to remove after the data team update, it will become useless (NAVP-1285)
+    if (stop_area_c == unknown_column) {
+        stop_id_is_present = false;
+        stop_area_c = csv.get_pos_col("station_id");
+        for (const auto& object_code_map : data.object_codes) {
```

Comment:
I don't think we only want to associate object_codes when there is no `stop_id` column.

Reply:
Actually, when there is no `stop_id` column, we want to read `station_id` for retrocompatibility. 
Now, we have this code that work with `station_id` column. I just skip this part because it work for `NAV 1 id`. 
It is a preprocessing to create a temporary stop_area list (tmp_stop_area_map)

In any case, that's what I understand

Reply:
I updated the PR's header with id info